packages/react/tsconfig.json

```diff
@@ -1,14 +1,16 @@
 {
   "compilerOptions": {
+    "moduleResolution": "node",
+    "baseUrl": ".",
```

Comment:
I thought this was the default, but `tsup` & `vite` weren't picking up on `node_modules` until I added this.